src/components/media/index.tsx

```diff
@@ -1,14 +1,6 @@
+import { ImageProps } from '@/types/index'
 import Image from '@/components/image'
 
```

Comment:
Everything that we called 'display props' or 'component props' moves down to `@/types/`. These types serve two purposes:
* They define what the component should implement.
* They define what the drupal->next-build mapping should output.